sql/core/src/main/java/org/apache/spark/sql/expressions/javalang/typed.java

```diff
@@ -74,4 +71,40 @@
   public static <T> TypedColumn<T, Long> sumLong(MapFunction<T, Long> f) {
```

Comment:
we can also add average, which is already implemented.

Reply:
Its already there a bit higher up in the file